src/Illuminate/Support/Traits/Conditional.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Illuminate\Support\Traits;
+
+trait Conditional
+{
+    /**
+     * Apply the callback if the given "value" is true.
+     *
+     * @param mixed         $value
+     * @param callable      $callback
+     * @param callable|null $default
+     *
+     * @return mixed|$this
```

Comment:
Can you please use the same DocBlock style as we use in the rest of the framework? Also `mixed` can never be combined with something else. `$this` is already part of `mixed`. Thanks

Reply:
Sure thing. I just copy-and-pasted this from the implementation in `BuildsQueries`, but I'll update it.

Reply:
Although I think adding `mixed|$this` will provide significantly better code completion in IDEs—that's probably why it was there in the BuildsQueries DocBlock…